lib/ansible/modules/files/file.py

```diff
@@ -32,9 +32,9 @@
     aliases: [ dest, name ]
   state:
     description:
-    - If C(absent), directories will be recursively deleted, and files or symlinks will
-      be unlinked. Note that C(absent) will not cause C(file) to fail if the C(path) does
-      not exist as the state did not change.
+    - If C(absent), directories will be recursively deleted when C(recurse=True), and files 
+      or symlinks wil be unlinked. Note that C(absent) will not cause C(file) to fail if 
```

Comment:
Make this a new entry.
```suggestion
      or symlinks will be unlinked.
    - Note that C(absent) will not cause C(file) to fail if the C(path) does not exist as the state did not change.
```